subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AndroidIncrementalExecutionPerformanceTest.groovy

```diff
@@ -48,12 +48,13 @@ class AndroidIncrementalExecutionPerformanceTest extends AbstractIncrementalExec
         runner.args.addAll(["--no-build-cache", "--no-scan"])
         // use the deprecated property so it works with previous versions
         runner.args.add("-D${StartParameterBuildOptions.ConfigurationCacheProblemsOption.DEPRECATED_PROPERTY_NAME}=warn")
+        runner.warmUpRuns = 20
```

Comment:
:+1: this raised confidence quite a bit for both SantaTracker and NowInAndroid. Moreover these tests are quick enough for this to not slow down CI feedback